tests/reading-tests.md

```diff
@@ -0,0 +1,397 @@
+
+It's a comment indicating an "Arrange" step and a `const` variable `num` assigned to the number `9`.
+
+Leila is an overthinker, and they could dig deeper into what this line means. However, this time, they will choose to make only one theory: the variable `num` will be used later in the test.
+
+### Act
+
+We can move on to reading the next set of lines:
+
+<!-- prettier-ignore-start -->
+```js
+    // Act
+    const result = fizzBuzz(num);
+```
+<!-- prettier-ignore-end -->
+
+Leila recognizes this as the "Act" step in a test, where the behavior we're testing (a `fizzBuzz` function) should be "acted on," or invoked.
+
+Here, we are invoking `fizzBuzz`, passing in `num`, and saving the return value into a `const` variable named `result`.
+
+### Assert
+
+Let's dive into the final line, which features some more unfamiliar syntax.
+
+<!-- prettier-ignore-start -->
+```js
+    // Assert
+    expect(result).toEqual('Fizz');
+```
+<!-- prettier-ignore-end -->
+
+Leila knows that the assert step is the step that determines whether a test passes or fails.
+
+They've seen the `test` block include the string "returns Fizz on multiples of 3," which Leila suspects is relevant to the Assert step of this test.
+
+Reading the line from left-to-right, it seems like `expect(result)` is also a function call. The function's name is `expect` and we're passing in `result`.
+
+However, the line doesn't stop there. Leila sees `.toEqual('Fizz')`. Recalling other JavaScript syntax, Leila suspects that this is a function call, where the function is named `toEqual` and the argument is `'Fizz'`.
+
+It looks like this function call operates from an object. That object must be `expect(result)`.
+
+### Reading it Out Loud
+
+Leila can now take a step back and read this line overall. They notice a sentence that the code forms: "Expect the value of `result` to equal `'Fizz'`."
+
+The test will pass if `result`'s value is `'Fizz'`, and fail if it isn't.
+
+## Multiple Tests in One Describe Block
+
+Leila just can't catch a break! After reading through the first test, Nadia is prompting Leila to continue reading.
+
+However, Leila can use this opportunity to possibly validate or disprove some of their theories.
+
+<!-- prettier-ignore-start -->
+```js
+describe('fizzBuzz', () => {
+  test('returns Fizz on multiples of 3', () => {
+    // Arrange
+    const num = 9;
+
+    // Act
+    const result = fizzBuzz(num);
+
+    // Assert
+    expect(result).toEqual('Fizz');
+  });
+
+  test('returns Buzz on multiples of 5', () => {
+    expect(fizzBuzz(25)).toEqual('Buzz');
+  });
+});
+```
+<!-- prettier-ignore-end -->
+
+The first thing Leila notices is that the next test is a second `test` block, located in the same `describe` block.
+
+There aren't commas between the tests, because these tests are not in an object with comma-separated key-value pairs, nor are they in a comma-separated array.
+
+The `test` block continues to be, ultimately, a function call: `test(..., ...);`
```

Comment:
This is a different test block, so we can't say that it continues to be anything, since we are just looking at it now. However, it can be something _also_, with respect to the first test.
```suggestion
This `test` block also seems to be, ultimately, a function call: `test(..., ...);`
```

Reply:
I like Ansel's suggested change but am confused about the "also". It isn't clear to me what text ahead of this the also is referring to. Unless I'm missing something obvious, I'd vote we remove "also"

Reply:
The "also" is referring to the first `test` block, which was mentioned to be ultimately a function call to `test()`. Still gonna remove the "also" to eliminate that confusion

```suggestion
This `test` block seems to be, ultimately, a function call: `test(..., ...);`
```